src/Features/Core/Portable/QuickInfo/QuickInfoService.cs

```diff
@@ -16,13 +18,13 @@ public abstract class QuickInfoService : ILanguageService
         /// <summary>
         /// Gets the appropriate <see cref="QuickInfoService"/> for the specified document.
         /// </summary>
-        public static QuickInfoService GetService(Document document)
+        public static QuickInfoService? GetService(Document? document)
```

Comment:
Do we _really_ want to be able to pass a null document here? It seems like more descriptive code to have this not accept null.

Reply:
So here's the rub: if somebody is today passing in null and they upgrade to a library with these annotations, do we want to flag it, even though it might work?